src/views/dashboard/Support.vue

```diff
@@ -142,21 +223,14 @@ export default {
 </script>
 
 <style scoped>
-.productList {
-  @apply space-y-2;
+.products {
+  @apply flex space-x-2;
 }
 
 /* list item */
-.productList__item {
-  @apply grid auto-rows-auto gap-y-4 bg-white text-gray-500 border-l-8 border-gray-400 rounded-md w-full p-5;
+.products .product {
```

Comment:
I try to keep the styling within the component, so `.product` could be within the Product component with scoped style